bin/ikestrel

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+# Executing a hunt in an interactive CLI
+# Usage: `ikestrel [-v] [--debug]`
+
+import argparse
+import cmd
+import logging
+
+from tabulate import tabulate
+
+from kestrel.session import Session
+from kestrel.codegen.display import DisplayBlockSummary, DisplayDataframe
+from kestrel.exceptions import KestrelException
+from kestrel.utils import add_logging_handler, clear_logging_handlers
+
+
+CMDS = [  # command_no_result from kestrel.lark
+    "APPLY",
+    "DISP",
+    "INFO",
+    "SAVE",
+]
+
+
+def display_outputs(outputs):
+    for i in outputs:
+        if isinstance(i, DisplayBlockSummary):
+            print(i.to_string())
+        elif isinstance(i, DisplayDataframe):
+            data = i.to_dict()["data"]
+            print(tabulate(data, headers="keys"))
+        else:
+            print(i.to_string())
+
+
+class Cli(cmd.Cmd):
+    prompt = "> "
+
+    def __init__(self, session: Session):
+        self.session = session
+        self.buf = ""
+        super().__init__()
+
+    def default(self, line: str):
+        try:
+            outputs = self.session.execute(line)
+            display_outputs(outputs)
+        except KestrelException as e:
+            print(e)
+
+    def completenames(self, text, *ignored):
```

Comment:
Is this a virtual method to override? I could not find it at https://docs.python.org/3/library/cmd.html